.env.example

```
@@ -152,6 +152,13 @@ calendar_default_enabled=false
 calendar_default_modes=auto,weather,token,token-always
 calendar_default_refresh_seconds=90
 
+# Nearby-flight encounters are persisted locally for on-demand day/week/month
+# statistics and record screens. The database stays in the ignored cache tree.
+flight_statistics_db=cache/flight_statistics.sqlite3
```

dotenv-linter (4.0.0)

[warning] [LowercaseKey] The flight_statistics_db key should be in uppercase

(LowercaseKey)